test/transaction_test.rb

```diff
@@ -29,4 +37,16 @@ def test_transaction_has_a_result
     assert_equal "success", transaction.result
   end
 
+  def test_find_invoice_returns_an_invoice_for_associated_transaction
+    result = @transaction_repository.transactions[9].invoice
+
+    assert_equal 3, result.customer_id
+  end
+
+  private
+
+  def fixtures_directory
+    File.expand_path('../../data/fixtures', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.